mercat/wasm/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
What's the legality of subtree licenses in the repository (we already have a license file at the root)?  Or is this strictly a duplication of the root `LICENSE` file for artifacts (in which case I would have the pipeline copy in the root file so that it can be updated in a single place).

Reply:
Yeah having pipeline copy it is better (I copied it since some of the packaging tools were complaining about the lack of license in the subtrees).

Reply:
Removed the LICENSE file from subdirectories, the scripts will copy them as needed.